operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -26,96 +26,17 @@ The Cray CLI must be configured on the node where the commands are being run. Se
 1. (`ncn-mw#`) Identify the NCN image to be modified.
 
     If this procedure is being done as part of a CSM upgrade, then the documentation which linked to this procedure will have
-    provided instructions for how to set the `ARTIFACT_VERSION` variable.
+    provided instructions for how to set the `K8S_IMS_IMAGE_ID` variable.
```

Comment:
Since this is meant to be a generic procedure that works for either k8s or storage NCNs, would it be better to make the variable name generic? `NCN_IMAGE_ID`?

Reply:
Yeah, agreed.  I changed it to `NCN_IMS_IMAGE_ID`.